mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -5265,7 +5517,7 @@ struct GridwiseGemmRewritePattern : public OpRewritePattern<miopen::GridwiseGemm
         max_lds_align *
         math_util::integer_divide_ceil<int64_t>(MPerBlock, max_lds_align);
     a_block_space = math_util::integer_least_multiple(KPerBlock * AlignedMPerBlock,
-                                                 max_lds_align);
+                                                 max_lds_align) * KPack;
```

Comment:
This is the C++ logic for A/B tensor in LDS with [KPerBlock, MPerBlock, KPack] layout

https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/a4f24233e51854c4b5cb7d75637fa0f235f78f8e/composable_kernel/include/tensor_operation/gridwise_gemm_xdlops_v2r3.hpp#L409-L439